rplidar.go

```diff
@@ -40,19 +45,18 @@ var (
 type Rplidar struct {
 	resource.Named
 	resource.AlwaysRebuild
-	mu                      sync.Mutex
-	close                   bool
-	device                  rplidarDevice
-	nodes                   gen.Rplidar_response_measurement_node_hq_t
-	nodeSize                int
```

Comment:
[note] removed magic numbers from new that remain unchanged during operation and instead have them as constants.